nym-wallet/src/components/Modals/LoadingModal.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Box, CircularProgress, Modal, Stack, Typography } from '@mui/material';
+
+const modalStyle = {
+  position: 'absolute' as 'absolute',
```

Comment:
are you sure about this cast `'absolute' as 'absolute'` ?

Reply:
I managed to remove it :100: